moto/ec2/models/instances.py

```diff
@@ -538,6 +535,12 @@ def get_instance(self, instance_id):
         raise InvalidInstanceIdError(instance_id)
 
     def add_instances(self, image_id, count, user_data, security_group_names, **kwargs):
+        location_type = "availability-zone" if "placement" in kwargs and kwargs["placement"] else "region"
```

Comment:
```suggestion
        location_type = "availability-zone" if kwargs.get("placement") else "region"
```
This is logically equivalent, I believe, and more readable